content/actions/using-workflows/workflow-commands-for-github-actions.md

```diff
@@ -496,12 +496,15 @@ on: push
 jobs:
   secret-generator:
     runs-on: ubuntu-latest
+    outputs:
+      handle: ${{ steps.generate-secret.outputs.handle }}
```

Comment:
```suggestion
      handle: {% raw %}${{ steps.generate-secret.outputs.handle }}{% endraw %}
```

We should add `{% raw %}` and `{% endraw %}` tags here so the [bracketed content renders properly](https://docs.github.com/en/contributing/style-guide-and-content-model/style-guide#code-blocks) 🙂